core/src/components/datetime/test/basic/datetime.e2e.ts

```diff
@@ -408,3 +408,30 @@ test('datetime: md highlight should not clip at start or end of month', async ({
 
   await expect(datetime).toHaveScreenshot(`date-highlight-end-of-month-${page.getSnapshotSettings()}.png`);
 });
+
+test.describe('datetime: dark mode', () => {
```

Comment:
This is a fix for `ion-modal`, so this test should be in the `ion-modal` tests instead. Additionally, we should be able to pass text into the modal instead of a datetime.